apps/prairielearn/src/pages/instructorFileEditor/instructorFileEditor.ts

```diff
@@ -91,28 +70,9 @@ router.get(
 
     const fullPath = paths.workingPath;
     const relPath = paths.workingPathRelativeToCourse;
-    const fileEdit: FileEdit = {
-      userID: res.locals.user.user_id,
-      authnUserId: res.locals.authn_user.user_id,
-      courseID: res.locals.course.id,
-      coursePath: paths.coursePath,
-      dirName: path.dirname(relPath),
-      fileName: path.basename(relPath),
-      fileNameForDisplay: path.normalize(relPath),
-      aceMode: modelist.getModeForPath(relPath).mode,
-      alertChoice: false,
-      didSave: false,
-      didSync: false,
-    };
-
-    debug('Read from db');
-    await readDraftEdit(fileEdit);
 
     debug('Read from disk');
```

Comment:
Hot take: we could probably drop `debug` here? What do you think @mwest1066?

Reply:
Yep, I'm on board.